package.json

```diff
@@ -56,7 +56,7 @@
     "lodash.orderby": "^4.6.0",
     "lodash.trim": "^4.5.1",
     "lodash.uniqby": "^4.7.0",
-    "material-design-icons": "3.0.1",
+    "material-design-icons-iconfont": "5.0.1",
```

Comment:
@Bitcoinera the official material icons library is too big and takes too long to download - so it was causing our build system to fail. i switched it for this one which just downloads the files we need (not the svgs, pngs, jpgs, etc...)